x-pack/plugins/lists/common/schemas/common/schemas.test.ts

```diff
@@ -98,15 +50,16 @@ describe('Common schemas', () => {
       expect(message.schema).toEqual(payload);
     });
 
-    test('it should contain 2 keys', () => {
+    test('it should contain same amount of keys as enum', () => {
       // Might seem like a weird test, but its meant to
       // ensure that if operator is updated, you
       // also update the operatorEnum, a workaround
       // for io-ts not yet supporting enums
       // https://github.com/gcanti/io-ts/issues/67
       const keys = Object.keys(operator.keys);
+      const enumKeys = Object.keys(OperatorEnum);
 
-      expect(keys.length).toEqual(2);
+      expect(keys.length).toEqual(enumKeys.length);
```

Comment:
It would be nice but not essential to check that keys and enumKeys have the same keys in them

Reply:
Thanks @marshallmain ! I went ahead and updated the tests to do just that.